plio/static/plio/docs/download_csv_README.md

```diff
@@ -81,23 +81,20 @@ This section will clarify what each of the `.csv` files in the folder contains:
 
   - `session_id`: the unique identifier for the session
 
-  - `user_identifier`: the unique identifier for the user associated with this session
+  - `user_identifier`: the unique identifier for the user associated with this session. To preserve user privacy, this field would not contain any Personally Identifiable Information (PII) and would instead be a hashed value. However, the same user will have the same hashed value across different plios. So, you can still safely identify user trends across plios without harming user privacy. However, if you are on the organisational plan and want to map the user ids to your beneficiaries, we provide a way for you to access the true identities of the users interacting with your plio. Read about it [here](https://docs.plio.in/plio-for-teams/).
```

Comment:
Should we rename `hashed value` to `coded value` to allow more people to understand what we mean to say?